DP1.10081.002/Table.joining.md

```diff
@@ -0,0 +1,6 @@
+|Table 1|Table 2|Join by field(s)|
```

Comment:
Include linkages to the soil data product as well. You don't need to include every table, though, e.g. if it makes the most sense, it's fine to include only the joining field for the soil collection table here, and note in the Standard calculations section that other tables can be linked via the collection table.

Reply:
I have added linkages to the soil data product, and for aquatics, the relevant tables in mmg to get more field data. As mentioned above, it is difficult to go beyond this as MCT does not include parentID.